Sample Data/Custom/BetterMTDDeviceLog_CL.csv

```diff
@@ -0,0 +1,30 @@
+﻿TenantId,SourceSystem,TimeGenerated,RawData,"BuildNumber_s","Manufacturer_s","DeviceId_d","DeviceUDID_g","DeviceOS_s","CompanyId_d","DevicePlatform_s","LastReported_t","LocationID_d","CompanyName_s","UserEmail_s","IsDeleted_d","ThreatLevel_s","ThreatScore_d","AddedDate_t","AgentVersion_s",Type
```

Comment:
@preetikr, is there a requirement to have sample data in .json file?

Reply:
@aymenim, please provide json format sample data. According to the guideline, Json format sample is required for rest API connecotr.